conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md

```diff
@@ -44,28 +44,32 @@ in place by a project (for reports made to a project) or the CPC (in the
 case of reports relating to spaces within the scope of the foundation versus a project).
 In exceptional cases where a reporter wishes to challenge the response from the CPC or
 project, or does not feel comfortable reporting through the standard processes, an escalation
-report can be made to coc-escalation@openjsf.org.
+report can be made to report@openjsf.org.
 
 All escalations will be reviewed and investigated by the Code of Conduct Panel (CoCP)
 and will result in a response that is deemed necessary and appropriate to the circumstances.
 The Code of Conduct panel is obligated to maintain confidentiality with regard
 to the reporter of an escalation.
 
-### Code of Conduct panel
+### Code of Conduct Panel
 
-The Code of Conduct panel is a cross-foundation team who have been trained in moderation
+The Code of Conduct Panel is a cross-foundation team who have been trained in moderation
 and code of conduct issues. The makeup of this team is as follows:
 
-* The Foundation Executive director
-* 1 member from the Board
-* 1 member from the CPC
-* 1 member of the OpenJS marketing committee
-* each top-level project may optionally provide 1 member
-* the non top-level projects may optionally provide 1 member
-* optionally 1 invited outside expert as agreed by the other CoCP members.
+* OpenJS Foundation Executive Director - Robin Ginn (@rginn)
+* OpenJS Foundation Marketing Lead - Kylie Wagar-Dirks (@kyliewd)
+* 3 members selected by the Cross Project Council
+  - TBD
+  - TBD
+  - TBD
 
-Members of the CoCP team will be added for a 1-year term and will be re-confirmed on
-a yearly basis.
+#### Nomination
+
+Code of Conduct Panel team members are CPC Members who self-nominate or are community
+members nominated by the CPC. CoCP team members must be approved by the CPC and re-confirmed
+every year. If there are no objections after seven days, the nomination is automatically
+accepted. If there are objections to a specific nomination, then a simple majority vote of
+the CPC in favor of the nomination is required.
 
```

Comment:
```suggestion
The OpenJS Foundation's Code of Conduct team members are comprised of OpenJS staff (as outlined above),
CPC Members or community members who self-nominate and are then voted on by CPC Voting Members using
ranked choice voting method. Nominations will be open for two weeks. Elections will be open for two weeks
and will happen on a  yearly basis.
```

Reply:
OMG NOT MORE VOTING!!

Reply:
More seriously, we should use consensus to agree on a panel that we feel is a good fit for the mission and diverse enough. We shouldn't be voting on this, imho.

Reply:
@tobie there is still voting. Maybe the phrasing was just not the best, but there is definitely still voting 🙃

Reply:
Voting if we don't come to consensus seems OK. It's vaulting by default that's problematic.

Reply:
Not sure I'm then understanding what you mean by voting?

If someone gets nominated the voting members still need to give +1 to the candidate. Please correct me if I misunderstood that.

Reply:
> If someone gets nominated the voting members still need to give +1 to the candidate.

That's not really voting or electing someone. It is manifesting consent.

Reply:
I'd much rather we use the CoC team as an experiment in defining clear requirements for a good CoC team and then picking the right folks through consensus, rather than punting to elections.

Reply:
I agree with @tobie and experimenting with CoC

Reply:
But if we limit the amount of people to 3, which I think is a good idea (dont need to herd cats when dealing with a pressing CoC violation) and there are more than 3 people running, how do we "reach consensus"?

Reply:
> But if we limit the amount of people to 3, which I think is a good idea (dont need to herd cats when dealing with a pressing CoC violation) and there are more than 3 people running, how do we "reach consensus"?

In Node.js, having more eyes on a sensitive issue was always correlated with better outcomes and oftentimes having 5-6 people on a call provided a lot of different valuable perspectives that helped frame issues. Not coming to an agreement between moderators about a CoC violation happened exactly 0 times.

Moderators also don't _need_ to vote when taking actions the reason consensus is often reached even though the Node.js process for moderation doesn't actually require it and allows every single moderator to take immediate action (like blocking). In the few times we voted it was because we wanted to make sure everyone was heard when policy/repeat issues were involved. When we arranged calls to talk about specific issues it was because the moderator didn't feel comfortable taking action without consulting and getting feedback.

As a concrete example: several times I wrote emails or comments regarding CoC violations or escalating situations and feedback from other moderators really helped improve those emails and the outcomes we got. Sometimes no one was online and I _had_ to act by myself and people would come up with really good improvements after the fact which always made me go "I wish we had more people in the moderation team".

Reply:
```suggestion
#### Code of Conduct Team Composition

The CPC is responsible for forming the Code of Conduct Team. It should gather advisory input from the current Code of Conduct team to do so.

It is the CPC's duty to ensure that the Code of Conduct Team is as representative and diverse as possible. In particular, the CPC should pay attention to:

- a strong representation of underrepresented minorities,
- geographic diversity,
- project diversity.

To form the Code of Conduct Team, the CPC must run a call for nominations at least yearly and commit to proactively reaching out to constituencies who are not currently well represented within the Code of Conduct Team. Members from the community can self-nominate or nominate their peers during the call period.

As part of their nomination, nominees must agree to abide by the terms of the [COC_TEAM_MEMBER_EXPECTATIONS.md](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/COC_TEAM_MEMBER_EXPECTATIONS.md). (TODO: pull questions from https://github.com/openjs-foundation/cross-project-council/issues/985 to write that up.)

The CPC must then run a consensus-driven process to decide on the composition of the Code of Conduct Team, bearing in mind its duty to create a diverse and representative group as defined above. This process must be run in a private CPC session in order to protect the privacy of the nominees. (TODO: decide whether CPC nominees can attend it or not.)

_Substantiated_ objections by CPC members for the inclusion of nominees in the Code of Conduct Team (for example for serious violations of the CoC, a documented history of hate speech, etc.), cannot be overruled by a vote.

There is not a strict number of seats in the Code of Conduct Team. However it is expected that the Code of Conduct Team is composed of at least 5 members (including the ED and Events coordinator), and no more than 12 members total.
```

Reply:
FWIW I just want to restate what I shared in today's working session and what I've said before:

I highly recommend that we follow a process where voting isn't necessary, and people who are nominated or self-nominate are selected based off of complete consensus - in effect, zero -1s - from the CPC membership. Whether this is voting membership or full CPC membership, I don't particularly mind. If there are _only_ +1s, the person is added to moderation. One minor addition, if there *is* a CPC member who wants to vote -1 but does not want to do so publicly, they should be able to reach out to the Chair and share that -1 with a reason, and have the Chair proxy that -1.

I do not believe that we need to have dedicated sessions in CPC meetings for this selection to happen. This has happened in GitHub Issues in Node.js for years now without significant issue (there was _one_ time where someone was on an extended vacation and missed a deadline to -1 when they wanted to -1). 


Specifically noting some things from @tobie's suggestion:


> As part of their nomination, nominees must agree to abide by the terms of the [COC_TEAM_MEMBER_EXPECTATIONS.md](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/COC_TEAM_MEMBER_EXPECTATIONS.md). (TODO: pull questions from https://github.com/openjs-foundation/cross-project-council/issues/985 to write that up.)


IMO this document will be no different than the existing [Membership Expectations](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/MEMBER_EXPECTATIONS.md) document. If it's _more_ restrictive, I'd prefer Membership Expectations to be updated to be more restrictive.

> _Substantiated_ objections by CPC members for the inclusion of nominees in the Code of Conduct Team (for example for serious violations of the CoC, a documented history of hate speech, etc.), cannot be overruled by a vote.

If there is something else that follows into "etc", I would very clearly define it here. Leaving an open end in a space like this results in gray areas that often get ignored or brushed off, which is not a situation I want us to be in for something as important as moderation.

Also, how do you define Substantiated? If someone's putting out white supremacist or TERF dog whistles but not explicitly saying things, would that be considered substantiated? 

> There is not a strict number of seats in the Code of Conduct Team. However it is expected that the Code of Conduct Team is composed of at least 5 members (including the ED and Events coordinator), and no more than 12 members total.

I'd like to request that this also include a section about company affiliation to disallow over-representation of single corporate entities.

Reply:
Thanks for your insightful comments, @bnb, and apologies for missing today's call.

> I highly recommend that we follow a process where voting isn't necessary, and people who are nominated or self-nominate are selected based off of complete consensus - in effect, zero -1s - from the CPC membership. Whether this is voting membership or full CPC membership, I don't particularly mind. If there are _only_ +1s, the person is added to moderation. One minor addition, if there _is_ a CPC member who wants to vote -1 but does not want to do so publicly, they should be able to reach out to the Chair and share that -1 with a reason, and have the Chair proxy that -1.

As mentioned earlier, I would prefer that we operated (across the organization in general, not just here) with the intent of forming teams to fulfill clearly stated missions, rather than through self-nomination, approval, or voting, as these create a false sense of democracy while limiting candidates to those who are confortable self-nominating.

> Specifically noting some things from @tobie's suggestion:
> > As part of their nomination, nominees must agree to abide by the terms of the [COC_TEAM_MEMBER_EXPECTATIONS.md](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/COC_TEAM_MEMBER_EXPECTATIONS.md). (TODO: pull questions from #985 to write that up.)
> 
> IMO this document will be no different than the existing [Membership Expectations](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/MEMBER_EXPECTATIONS.md) document. If it's _more_ restrictive, I'd prefer Membership Expectations to be updated to be more restrictive.

That's a fair point. +1

> 
> > _Substantiated_ objections by CPC members for the inclusion of nominees in the Code of Conduct Team (for example for serious violations of the CoC, a documented history of hate speech, etc.), cannot be overruled by a vote.
> 
> If there is something else that follows into "etc", I would very clearly define it here. Leaving an open end in a space like this results in gray areas that often get ignored or brushed off, which is not a situation I want us to be in for something as important as moderation.

I understand your point. However, we can't reasonably provide a complete list and examples are useful. So my suggestion here would be to find a list of examples, and be clear about its lack of completeness. 

> Also, how do you define Substantiated? If someone's putting out white supremacist or TERF dog whistles but not explicitly saying things, would that be considered substantiated?

I should have said "motivated" rather than "substantiated." Would that alleviate your concern?

> > There is not a strict number of seats in the Code of Conduct Team. However it is expected that the Code of Conduct Team is composed of at least 5 members (including the ED and Events coordinator), and no more than 12 members total.
> 
> I'd like to request that this also include a section about company affiliation to disallow over-representation of single corporate entities.

I feel like this is captured in the requirement to "create a diverse and representative group." I'm personally more concerned about project homogeneity than corporate homogeneity for a CoC group (I'd have different concerns for the board, for example). WDYT?

Reply:
> limiting candidates to those who are confortable self-nominating.

Are you aware of anyone who wanted to be a moderator and was OK with being nominated but was not willing to self-nominate? I'm asking to learn it sounds very counter-intuitive since it's a role that requires you to have unpleasant conversations and put yourself in the middle of conflict.

Reply:
My point is only that having a diverse group requires outreach.